subset/alliedSwitch/.gitignore

```diff
@@ -0,0 +1,2 @@
+tmp/*

```

Comment:
ok, this makes sense for now, I will update and revert, one of those chicken and egg situations! there is a caveat though as the switch module future edition will also support cisco switches, idea is to use a command line arg for selecting cisco/allied

Reply:
I'm not sure what you mean by reverting or chick-and-egg.  I was just suggesting simply moving files to a different directory.

If you want to make the command-line approach that's fine too, but then it wouldn't be in the subset/alliedSwitch directory either. Then it would just be subset/switches and go from there.

Reply:
I also think it makes sense to move it to subset/switches. Eclipse also has "Extract method" so Ed can easily refactor the code into smaller chunks.

Reply:
complete